tests/integration/migrator/migrator_declare_cheatcode_test.py

```diff
@@ -0,0 +1,46 @@
+import re
+from pathlib import Path
+
+import pytest
+
+from protostar.commands.test.test_environment_exceptions import CheatcodeException
+from protostar.migrator import Migrator
+
+
+async def test_declare_contract(
```

Comment:
Can we include a test declare -> deploy the declared contract on devnet

Reply:
Nope.`deploy` cheatcode doesn't exist. Deploy syscall would work within the Protostar's devnet which doesn't make sense. We need to add `call` migration cheatcode so that a user can trigger a code that uses a `deploy` syscall.

Reply:
Can't we make call using for example starknet.py or something?

Reply:
Duplicated thread. Resolving.